### ISSUE TITLE
Examples do not build

### ISSUE THREAD

**Issue by iosdev-republicofapps** [ButterKit/Butter#75]
Trying to compile any of the examples in the examples workspace gives the following error:
```
Butter/BTRView.m:130:16: No visible @interface for 'NSView' declares the selector 'actionForLayer:forKey:'
```
Any ideas?

Is this framework still under active development and I'd be happy to help if any help is needed.  :-)

Thanks!

**Comment by jwilling** [comment]
@indragiek Happen to know why `-actionForLayer:forKey:` was removed from `NSView` in 10.12? I haven't kept up with OS X developer release notes, not sure if you have either.

**Comment by indragiek** [comment]
@jwilling That's a `CALayerDelegate` method isn't it? https://developer.apple.com/reference/quartzcore/calayerdelegate/2097264-actionforlayer

**Comment by jwilling** [comment]
@indragiek Yeah you're right. I guess what happened then is that `NSView` was declared as conforming to `CALayerDelegate` but no longer is.

**Comment by johndpope** [comment]
ok - stupid hack solution / unsupported / don't call me if and when this breaks / blows up. 

in header 
```objc
@interface HackView : NSView <CALayerDelegate>
@end

@interface BTRView : HackView <CALayerDelegate>
@end

in implementation 
@implementation HackView
@end


this 
     [NSApp sendAction:action.action to:action.target]; -> 
to this
     [NSApp sendAction:action.action to:action.target from:self];
```
